### PR TITLE
Update initial price.php at first loading of price page when multiprice on

### PR DESCRIPTION
**Pull request by frederic34**

### CODE REVIEW

File: htdocs/product/price.php [hunk 1]
```diff
@@ -1439,7 +1440,12 @@ function on_change() {
 
     		// Il doit au moins y avoir la ligne de prix initial.
     		// On l'ajoute donc pour remettre a niveau (pb vieilles versions)
-    		$object->updatePrice($object->price, $object->price_base_type, $user, $newprice_min);
+    		//$object->updatePrice($object->price, $object->price_base_type, $user, $newprice_min);
+            if (! empty($conf->global->PRODUIT_MULTIPRICES)) {
+                $object->updatePrice($object->multiprices[1], $object->multiprices_base_type[1], $user, $object->multiprices_tva_tx[1], $object->multiprices_min[1], 1);
```
Comment (hregis) [inline htdocs/product/price.php:1445]: @frederic34 
Why is the value 1 selected in the "multiprices" array?
`$object->multiprices[1]`

Comment (frederic34) [reply htdocs/product/price.php:1445]: the first price level

Comment (hregis) [reply htdocs/product/price.php:1445]: @frederic34 
in an array the first is 0
`$object->updatePrice($object->multiprices[0], $object->multiprices_base_type[0], ...`

Comment (frederic34) [reply htdocs/product/price.php:1445]: 2018-02-05 - firefox developer edition

Comment (hregis) [reply htdocs/product/price.php:1445]: @frederic34 
arf ! sorry ! ;-)

Comment (frederic34) [reply htdocs/product/price.php:1445]: a page where prices are updated silently....

**Pull request merged**